codes/classical/properties/Low Density Parity Check (LDPC)

```diff
@@ -0,0 +1,33 @@
+
+code_id: LDPC
+
+name: 'Asymptotically Good LDPC Convolutional Codes Based on Protographs'
+introduced: '\cite{doi:10.1070/RM1997v052n06ABEH002155}'
+
+
+Description:
+- 'LDPC or Low Density Parity Check codes: A (N, J, K) binary code which its parity check matrix, H, has N columns, J ones in each column, K ones in each row and zeros elsewhere. This code can be built from a sparse bipartite graph (reffered to asTanner graph). (details in notes)'
```

Comment:
Description is a paragraph format, not a list. Please check template.yml carefully for other formatting guidelines.

Reply:
The paper was about a tail-biting convolutional LDPC code, so I thought first I describe LDPC, then convolutional LDPC and then the code in the paper (as best as I could).